src/Runner/SshRunner.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Castor\Runner;
+
+use Spatie\Ssh\Ssh;
+use Symfony\Component\Process\Process;
+
+/** @internal */
+final class SshRunner
+{
+    public function __construct(
+        private ProcessRunner $processRunner,
+    ) {
+    }
+
+    public function execute(
+        string $command,
+        string $path,
+        string $host,
+        string $user,
+        array $sshOptions = [],
+        ?bool $quiet = null,
+        ?bool $allowFailure = null,
+        ?bool $notify = null,
+        ?float $timeout = null,
+    ): Process {
+        $ssh = $this->configuration($host, $user, $sshOptions);
```

Comment:
this is weird for me to have a var called `ssh` but the method is `configuration`
I would call the method `buildSsh`

Reply:
Changed